src/SumoCoders/FrameworkCoreBundle/Controller/Tests/DatePickerController.php

```diff
@@ -30,15 +30,6 @@ public function indexAction()
                 )
             )
             ->add(
-                'date_example2',
-                DateType::class,
-                array(
-                    'widget' => 'text',
-                    'required' => false,
-                    'data' => new \DateTime(),
-                )
-            )
-            ->add(
                 'date_example3',
```

Comment:
Can you adjust the example numbers so they are sequential

Reply:
fixed 89a0a2e